### PR TITLE
Reseting thumb image

### LINKED ISSUE #25939: [Android, IOS, Mac] Unable to Remove Thumb Image in Slider Control
### Description

When the ThumbImageSource property of the Slider control is set to a valid image source (e.g., image_source.png), the image is displayed correctly on the thumb. However, if the ThumbImageSource is later set to null or an empty value (to remove the image), the image remains visible on the thumb, and it cannot be cleared
Image
 1. Click Thumb Image button
Image
 2.. Click Reset Button , thumb image is not removed
Image

 

### Steps to Reproduce

1. Click Thumb Image button
2. Click Reset button 
3. Observe thumb image is not removed.
4. [SliderSampleTest (2).zip](https://github.com/user-attachments/files/17811208/SliderSampleTest.2.zip)

### Link to public reproduction project repository

_No response_

### Version with bug

9.0.10 SR1

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

_No response_

### Affected platforms

Android, macOS, iOS

### Affected platform versions

_No response_

### Did you find any workaround?

_No response_

### Relevant log output

```shell

```

### PR DESCRIPTION
**Pull request by kubaflo**
### Issues Fixed

Fixes https://github.com/dotnet/maui/issues/25939

|Before|After|
|--|--|
|<video src="https://github.com/user-attachments/assets/aee5fe00-95fe-411b-9f6d-60994cf7562a" width="300px"/>|<video src="https://github.com/user-attachments/assets/8601f8bd-7af7-47e7-bd27-1b06c3d73d02" width="300px"/>|

### CODE REVIEW

File: src/Core/src/Platform/Android/SliderExtensions.cs [hunk 1]
```diff
@@ -63,6 +62,11 @@ public static async Task UpdateThumbImageSourceAsync(this SeekBar seekBar, ISlid
 				if (seekBar.IsAlive() && thumbDrawable != null)
 					seekBar.SetThumb(thumbDrawable);
 			}
+			else if (slider.Handler is SliderHandler sliderHandler)
+			{
+				seekBar.SetThumb(sliderHandler.defaultThumbDrawable);
```
Comment (PureWeen) [inline src/Core/src/Platform/Android/SliderExtensions.cs:67]: Can we load the default thumb drawable from the android resources? I'm sure the default drawable here can probably get pulled from the android resources

**Review by PureWeen — DISMISSED** [review]

File: src/Core/src/Platform/Android/SliderExtensions.cs (hunk 1 above)
Comment (kubaflo) [reply src/Core/src/Platform/Android/SliderExtensions.cs]: Yea, I was initially trying with this: `seekBar.SetThumb(context.GetDrawable(Resource.Drawable.abc_seekbar_thumb_material)), but I couldn't make it revert back to a default color, but I figured it out

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25939.cs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+		public override string Issue => "Unable to Remove Thumb Image in Slider Control ";
+
+		[Test]
+		[Category(UITestCategories.Slider)]
+		public void SliderShouldChangeThumbImageAndResetIt()
+		{
+			App.WaitForElement("ChangeThumbImageSourceButton");
+			App.Click("ChangeThumbImageSourceButton");
+			App.Click("ResetThumbImageSourceButton");
+			VerifyScreenshot();
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25939.cs:22]: Snapshots available in the latest build:
image

Could you commit the images?

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25939.cs (hunk 2 above)
Comment (kubaflo) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25939.cs:22]: Sure!

File: src/Core/src/Platform/Android/SliderExtensions.cs (hunk 1 above)
Comment (mattleibow) [reply src/Core/src/Platform/Android/SliderExtensions.cs]: "but I figured it out" - smart guy alert! 😆

Comment (kubaflo) [reply src/Core/src/Platform/Android/SliderExtensions.cs]: 😂 😂 😂

**Review by jsuarezruiz — APPROVED** [review]

**Review by jfversluis — APPROVED** [review]

**Pull request merged**